### PR TITLE
BugFix - [master] - LUA send_mail()

### PR DESCRIPTION
**Pull request by mafoo**
take call_uuid from the correct header
use smtp_from_name as part of the from address

### CODE REVIEW

File: resources/install/scripts/resources/functions/send_mail.lua [hunk 1]
```diff
@@ -92,19 +92,21 @@ if freeswitch then
 		xheaders = xheaders:sub(1,-2) .. '}'
 
 		local from = settins:get('email', 'smtp_from', 'var')
+		local from_name = settins:get('email', 'smtp_from_name', 'var')
+		local from_address = from_name .. "<" .. from .. ">"
```
Comment (ConnorStrandt) [inline resources/install/scripts/resources/functions/send_mail.lua]: This didn't seem to work on my install, it does however see to have the from header already set correctly.

Comment (markjcrane) [reply resources/install/scripts/resources/functions/send_mail.lua]: I think its because settins:get is spelled wrong and should be settings:get

**Comment by markjcrane** [comment]
It likely doesn't work because it is spelled wrong settins:get should actually be settings:get. I will fix it.

File: resources/install/scripts/resources/functions/send_mail.lua (hunk 1 above)
Comment (ConnorStrandt) [reply resources/install/scripts/resources/functions/send_mail.lua]: I tried it prior to submitting my pull the other day. Then again maybe I had a typo in mine too (my laptop doesn't like certain keys sometimes).

**Pull request merged**